chronos/asyncloop.nim

```diff
@@ -165,9 +165,15 @@ export asyncfutures2, timer
 
 # TODO: Check if yielded future is nil and throw a more meaningful exception
 
+const unixPlatform = defined(macosx) or defined(freebsd) or
```

Comment:
Aren't all these platforms covered by `defined(posix)`?

Reply:
Yes you are right, but according to https://github.com/nim-lang/Nim/blob/devel/compiler/platform.nim it is also supports MorphOS, SkyOS, Irix, AIX, QNX, Haiku, VxWorks and NintendoSwitch. And this compilation switch was introduced to avoid running chronos in non-tested environments.

Reply:
https://github.com/nim-lang/Nim/blob/devel/compiler/options.nim#L389-L394